cime/config/e3sm/config_grids.xml

```diff
@@ -2766,6 +2825,59 @@
       <map name="LND2ATM_SMAPNAME">cpl/gridmaps/360x720cru/map_360x720cru_to_ne1024np4_mono.190508.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne256np4.pg2" ocn_grid="oRRS18to6v3">
+      <!-- 13km atm /  18to6 ocean, so use bilin for state -->
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne256pg2/map_ne256pg2_to_oRRS18to6v3_nco.200212.nc</map>
+      <map name="ATM2OCN_SMAPNAME">cpl/gridmaps/ne256pg2/map_ne256pg2_to_oRRS18to6v3_bilin.200212.nc</map>
```

Comment:
I don't see cpl/gridmaps/ne256pg2/map_ne256pg2_to_oRRS18to6v3_bilin.200212.nc in the repo:
https://web.lcrc.anl.gov/public/e3sm/inputdata/cpl/gridmaps/ne256pg2/

Reply:
the ne256pg2, ne512pg2 andne1024pg2 cases are not finalized yet.  The are also missing initial condition files.   so those are not expected to work out of the box at this time.

Reply:
Can this be stated clearly in the description? It says 'new compsets'... So, only ne30 compset would work? What is the line to config this compset? I want to try.

Reply:
updated.